core/server/src/test/java/alluxio/worker/block/BlockMasterClientTest.java

```diff
@@ -49,8 +49,8 @@ public void unsupportedVersionTest() throws Exception {
 
     BlockMasterClient client =
         new BlockMasterClient(NetworkAddressUtils.getConnectAddress(
-            NetworkAddressUtils.ServiceType.MASTER_RPC, WorkerContext.getConf()),
-            WorkerContext.getConf());
+        NetworkAddressUtils.ServiceType.MASTER_RPC, WorkerContext.getConf()),
```

Comment:
I don't think that Checkstyle supports this differentiation in the indentations in different cases. I had a look at the Checkstyle rules for the Google Java style and the don't use ```forceStrictCondition```.